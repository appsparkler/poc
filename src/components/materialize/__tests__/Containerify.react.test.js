import React, {Component} from 'react'
import containerify from '../Containerify.react'
import renderer from 'react-test-renderer'

it('should correctly containerify a component', () => {
  class TestComponent extends Component {
    render() {
      return (
        <div>Test Component for containerify hoc component</div>
      )
    }
  }
  const ContainerifiedComponent = containerify(TestComponent)
  const wrapper = renderer.create(<ContainerifiedComponent />)
  const tree = wrapper.toJSON()
  expect(tree).toMatchSnapshot()
})
