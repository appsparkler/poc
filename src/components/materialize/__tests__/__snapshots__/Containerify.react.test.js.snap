// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly containerify a component 1`] = `
<div
  className="container"
>
  <div>
    Test Component for containerify hoc component
  </div>
</div>
`;
